### ISSUE TITLE
NullPointerException - clipboard.DiskOptimizedClipboard.getBlock

### ISSUE THREAD

**Issue by mibby** [boy0001/FastAsyncWorldedit#1094]
Paper dev 1552 (Spigot 1.12.2)
WorldEdit dev 3807
FAWE dev 1182

Config 
https://hastebin.com/pamotuzowi

```
[12:43:04] [Server thread/INFO]: player issued server command: //undo
[12:43:04] [Craft Scheduler Thread - 619/INFO]: WorldEdit: player (in "spawn"): /undo
[12:43:06] [Craft Scheduler Thread - 571/WARN]: java.lang.NullPointerException
[12:43:06] [Craft Scheduler Thread - 571/WARN]: 	at com.boydti.fawe.object.clipboard.DiskOptimizedClipboard.getBlock(DiskOptimizedClipboard.java:444)
[12:43:06] [Craft Scheduler Thread - 571/WARN]: 	at com.sk89q.worldedit.extent.clipboard.BlockArrayClipboard.getBlock(BlockArrayClipboard.java:188)
[12:43:06] [Craft Scheduler Thread - 571/WARN]: 	at com.sk89q.worldedit.function.pattern.ClipboardPattern.apply(ClipboardPattern.java:47)
[12:43:06] [Craft Scheduler Thread - 571/WARN]: 	at com.sk89q.worldedit.function.pattern.Pattern.next(Pattern.java:48)
[12:43:06] [Craft Scheduler Thread - 571/WARN]: 	at com.sk89q.worldedit.patterns.Pattern.apply(Pattern.java:47)
[12:43:06] [Craft Scheduler Thread - 571/WARN]: 	at com.sk89q.worldedit.EditSession.setBlock(EditSession.java:1120)
[12:43:06] [Craft Scheduler Thread - 571/WARN]: 	at com.sk89q.worldedit.EditSession.makeSphere(EditSession.java:2449)
[12:43:06] [Craft Scheduler Thread - 571/WARN]: 	at com.sk89q.worldedit.command.tool.brush.SphereBrush.build(SphereBrush.java:38)
[12:43:06] [Craft Scheduler Thread - 571/WARN]: 	at com.sk89q.worldedit.command.tool.BrushTool.act(BrushTool.java:488)
[12:43:06] [Craft Scheduler Thread - 571/WARN]: 	at com.sk89q.worldedit.command.tool.BrushTool.actPrimary(BrushTool.java:505)
[12:43:06] [Craft Scheduler Thread - 571/WARN]: 	at com.sk89q.worldedit.extension.platform.PlatformManager$7.run(PlatformManager.java:545)
[12:43:06] [Craft Scheduler Thread - 571/WARN]: 	at com.boydti.fawe.object.FawePlayer.lambda$runAction$3(FawePlayer.java:303)
[12:43:06] [Craft Scheduler Thread - 571/WARN]: 	at com.boydti.fawe.object.task.SimpleAsyncNotifyQueue.operate(SimpleAsyncNotifyQueue.java:25)
[12:43:06] [Craft Scheduler Thread - 571/WARN]: 	at com.boydti.fawe.object.task.AsyncNotifyQueue$1.run(AsyncNotifyQueue.java:15)
[12:43:06] [Craft Scheduler Thread - 571/WARN]: 	at org.bukkit.craftbukkit.v1_12_R1.scheduler.CraftTask.run(CraftTask.java:63)
[12:43:06] [Craft Scheduler Thread - 571/WARN]: 	at org.bukkit.craftbukkit.v1_12_R1.scheduler.CraftAsyncTask.run(CraftAsyncTask.java:52)
[12:43:06] [Craft Scheduler Thread - 571/WARN]: 	at com.destroystokyo.paper.ServerSchedulerReportingWrapper.run(ServerSchedulerReportingWrapper.java:22)
[12:43:06] [Craft Scheduler Thread - 571/WARN]: 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
[12:43:06] [Craft Scheduler Thread - 571/WARN]: 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
[12:43:06] [Craft Scheduler Thread - 571/WARN]: 	at java.lang.Thread.run(Thread.java:748)
[12:43:09] [Craft Scheduler Thread - 571/WARN]: java.lang.NullPointerException
[12:43:09] [Craft Scheduler Thread - 571/WARN]: 	at com.boydti.fawe.object.clipboard.DiskOptimizedClipboard.getBlock(DiskOptimizedClipboard.java:444)
[12:43:09] [Craft Scheduler Thread - 571/WARN]: 	at com.sk89q.worldedit.extent.clipboard.BlockArrayClipboard.getBlock(BlockArrayClipboard.java:188)
[12:43:09] [Craft Scheduler Thread - 571/WARN]: 	at com.sk89q.worldedit.function.pattern.ClipboardPattern.apply(ClipboardPattern.java:47)
[12:43:09] [Craft Scheduler Thread - 571/WARN]: 	at com.sk89q.worldedit.function.pattern.Pattern.next(Pattern.java:48)
[12:43:09] [Craft Scheduler Thread - 571/WARN]: 	at com.sk89q.worldedit.patterns.Pattern.apply(Pattern.java:47)
[12:43:09] [Craft Scheduler Thread - 571/WARN]: 	at com.sk89q.worldedit.EditSession.setBlock(EditSession.java:1120)
[12:43:09] [Craft Scheduler Thread - 571/WARN]: 	at com.sk89q.worldedit.EditSession.makeSphere(EditSession.java:2449)
[12:43:09] [Craft Scheduler Thread - 571/WARN]: 	at com.sk89q.worldedit.command.tool.brush.SphereBrush.build(SphereBrush.java:38)
[12:43:09] [Craft Scheduler Thread - 571/WARN]: 	at com.sk89q.worldedit.command.tool.BrushTool.act(BrushTool.java:488)
[12:43:09] [Craft Scheduler Thread - 571/WARN]: 	at com.sk89q.worldedit.command.tool.BrushTool.actPrimary(BrushTool.java:505)
[12:43:09] [Craft Scheduler Thread - 571/WARN]: 	at com.sk89q.worldedit.extension.platform.PlatformManager$7.run(PlatformManager.java:545)
[12:43:09] [Craft Scheduler Thread - 571/WARN]: 	at com.boydti.fawe.object.FawePlayer.lambda$runAction$3(FawePlayer.java:303)
[12:43:09] [Craft Scheduler Thread - 571/WARN]: 	at com.boydti.fawe.object.task.SimpleAsyncNotifyQueue.operate(SimpleAsyncNotifyQueue.java:25)
[12:43:09] [Craft Scheduler Thread - 571/WARN]: 	at com.boydti.fawe.object.task.AsyncNotifyQueue$1.run(AsyncNotifyQueue.java:15)
[12:43:09] [Craft Scheduler Thread - 571/WARN]: 	at org.bukkit.craftbukkit.v1_12_R1.scheduler.CraftTask.run(CraftTask.java:63)
[12:43:09] [Craft Scheduler Thread - 571/WARN]: 	at org.bukkit.craftbukkit.v1_12_R1.scheduler.CraftAsyncTask.run(CraftAsyncTask.java:52)
[12:43:09] [Craft Scheduler Thread - 571/WARN]: 	at com.destroystokyo.paper.ServerSchedulerReportingWrapper.run(ServerSchedulerReportingWrapper.java:22)
[12:43:09] [Craft Scheduler Thread - 571/WARN]: 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
[12:43:09] [Craft Scheduler Thread - 571/WARN]: 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
[12:43:09] [Craft Scheduler Thread - 571/WARN]: 	at java.lang.Thread.run(Thread.java:748)
```

**Comment by N0tMyFaultOG** [comment]
If this is still an issue, can you get a `/fawe debugpaste` ?

**Comment by N0tMyFaultOG** [comment]
Your ticket has been closed because it falls into one of the following categories:

- Your issue has been fixed in a newer version of FastAsyncWorldEdit for versions 1.8 - 1.12.2, get the latest build [here](https://ci.athion.net/job/FastAsyncWorldEdit/lastSuccessfulBuild/artifact/target/) matching the platform you need.

- Your issue targets minecraft version 1.13 or higher. This is the wrong repository then. Move to [this repository](https://github.com/IntellectualSites/FastAsyncWorldEdit) and open a bug report here.
BUT BEWARE: You should consider updating to 1.15.2, using the latest Fawe build for 1.15.2 (Build 255 when writing this, get it [here](https://ci.athion.net/job/FastAsyncWorldEdit-1.15/)) and reproducing the issue in a clean environment (fresh setup Spigot or Paper server).

- Your issue has been fixed in a build of FAWE for 1.15 or an issue exists on our new repository.

- You either didn't fill out the template or deleted it.

If none of the bullet points suits, head to the second one.
